src/guiFormSpecMenu.cpp

```diff
@@ -1040,6 +1040,11 @@ void GUIFormSpecMenu::parseSimpleField(parserData* data,
 		evt.KeyInput.PressedDown = true;
 		e->OnEvent(evt);
 
+#if USE_FREETYPE && IRRLICHT_VERSION_MAJOR == 1 && IRRLICHT_VERSION_MINOR < 9
+		if (use_freetype)
+			e->drop();
+#endif
```

Comment:
Oh, because the other block is.  I don't know why the original block had the version check, but I guess it was added for a reason.